fastlane/lib/fastlane/helper/sh_helper.rb

```diff
@@ -52,7 +59,7 @@ def self.sh_control_output(command, print_command: true, print_command_output: t
           message += "\n#{result}" if print_command_output
 
           error_callback.call(result) if error_callback
-          UI.shell_error!(message)
+          UI.shell_error!(message) if raise_shell_error
```

Comment:
Should we still print out the error message if `raise_shell_error` is set to true? I think it may be beneficial to at least be able to see the error in the logs still even if it won't bubble the error up 😇  Thoughts?